src/main/java/bio/terra/service/snapshot/flight/create/CreateSnapshotFireStoreDataStep.java

```diff
@@ -69,9 +70,8 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
             int numFilesSeen = 0;
             for (SnapshotMapTable mapTable : snapshotSource.getSnapshotMapTables()) {
                 for (SnapshotMapColumn mapColumn : mapTable.getSnapshotMapColumns()) {
-                    String fromDatatype = mapColumn.getFromColumn().getType();
-                    if (StringUtils.equalsIgnoreCase(fromDatatype, "FILEREF") ||
-                        StringUtils.equalsIgnoreCase(fromDatatype, "DIRREF")) {
+                    bio.terra.model.TableDataType fromDatatype = mapColumn.getFromColumn().getType();
```

Comment:
Don't need a fully qualified name here
```suggestion
                    TableDataType fromDatatype = mapColumn.getFromColumn().getType();
```

Reply:
ah yeah, my intellij has been doing this on imports and it's driving me nuts haha
I've tried changing the settings from recommendations like this,
https://intellij-support.jetbrains.com/hc/en-us/community/posts/206858289-auto-import-adding-full-package-name-to-the-line
But no luck